src/screens/AddVideo/AddVideo.component.tsx

```diff
@@ -3,13 +3,24 @@ import { Grid } from '@material-ui/core';
 import AddVideoFormik from './AddVideoFormik';
 
 interface OwnProps {
-  addVideo(videoUrl: string): void;
+  // addVideo(videoUrl: string): void;
+  addVideo(
+    videoUrl: string,
+    level: string | boolean,
+    rating: number | null
+  ): void;
 }
 
 const AddVideoComponent: FC<OwnProps> = ({ addVideo }) => {
   return (
     <Grid container direction="column" justify="center" alignItems="center">
-      <AddVideoFormik onSubmit={(values) => addVideo(values.youTubeLink)} />
+      <AddVideoFormik
+        // onSubmit={(values) => addVideo(values.youTubeLink)}
+        onSubmit={(values) =>
+          addVideo(values.youTubeLink, values.level, values.rating)
```

Comment:
i think that addVideo could take one argument of type AddVideoFormUrl, so that you can pass it directly to onSubmit